README.md

```diff
@@ -134,7 +134,7 @@ Give a ⭐️ if you like this project!
 
```

Comment:
![image](https://github.com/rachaouldbabaali/school-library/assets/93039359/dfba738e-c0dd-486e-8b55-a941f00eb4fd)

**[OPTIONAL]**

- [ ] It would be great if you make your commit messages capitalized to make your pull request more descriptive and professional 👍